src/opera/test/pge/tropo/test_tropo_pge.py

```diff
@@ -121,6 +126,16 @@ def test_tropo_pge_execution(self):
         # Check that a RunConfig for the SAS was isolated within the scratch directory
         expected_sas_config_file = join(pge.runconfig.scratch_path, 'test_tropo_config_sas.yaml')
         self.assertTrue(os.path.exists(expected_sas_config_file))
+        
+        # Check that the catalog metadata file was created in the output directory
+        expected_catalog_metadata_file = join(
+            pge.runconfig.output_product_path, pge._catalog_metadata_filename())
+        self.assertTrue(os.path.exists(expected_catalog_metadata_file))
+
+        # # Check that the ISO metadata file was created and filled in as expected
+        expected_iso_metadata_file = join(
+            pge.runconfig.output_product_path, pge._iso_metadata_filename())
+        self.assertTrue(os.path.exists(expected_iso_metadata_file))
```

Comment:
You should add a check here that the ISO file is fully populated and does not contain the template undefined error string
Reference: https://github.com/nasa/opera-sds-pge/blob/f13869f9c67a1b3451c3ca75fe10bcfcb09d96dc/src/opera/test/pge/dist_s1/test_dist_s1_pge.py#L179-L182